core/src/main/java/io/questdb/cairo/wal/CheckWalTransactionsJob.java

```diff
@@ -130,15 +134,25 @@ public boolean runSerially() {
 
     private boolean republishNotificationsFromTrackers() {
         engine.getTableTokens(tableTokenBucket, false);
+        long suspendedCount = 0;
+        long pendingTxnCount = 0;
         for (int i = 0, n = tableTokenBucket.size(); i < n; i++) {
             TableToken tableToken = tableTokenBucket.get(i);
             SeqTxnTracker tracker = engine.getTableSequencerAPI().getTxnTracker(tableToken);
-            if (!tracker.isSuspended() && tracker.getWriterTxn() < tracker.getSeqTxn()) {
+            if (tracker.isSuspended()) {
+                suspendedCount++;
+                continue;
+            }
+            long currTablePendingTxnCount = tracker.getSeqTxn() - tracker.getWriterTxn();
```

Comment:
We need both values in metrics, `seqTxn` and `writerTxn`. When `seqTxn` does not move it means no commits made or replication is down in case of a replica. If the difference goes up, then it means that WAL apply does not progress